packages/common/README/UTILITIES.md

```diff
@@ -31,9 +60,19 @@ import { permutations } from '@monkvision/common';
 console.log(permutations([1, 2, 3]));
 // Output : [[1, 2, 3], [1, 3, 2], [2, 1, 3], [2, 3, 1], [3, 1, 2], [3, 2, 1]]
 ```
-
 Returns an array containing all the possible permutations of the given array.
 
-| Param     | Type                                                   | Description                                         | Required | Default Value |
-|-----------|--------------------------------------------------------|-----------------------------------------------------|----------|---------------|
-| array     | array                                                  | The array to compute the permutations of.           | ✔️       |               |
+# Queue Utils
+## useQueue
+```typescript
+import { useQueue } from '@monkvision/common';
+
+function TestComponent() {
+  const queue = useQueue<string>((item) => console.log(item));
```

Comment:
Make a change in this statement for "process" function as function should return a Promise